apps/OpenSpace/main.cpp

```diff
@@ -995,6 +995,36 @@ void setSgctDelegateFunctions() {
     sgctDelegate.setStatisticsGraphScale = [](float scale) {
         sgct::Engine::instance().setStatsGraphScale(scale);
     };
+    sgctDelegate.setMouseCursor = [](WindowDelegate::Cursor mouse) {
+        static std::unordered_map<WindowDelegate::Cursor, GLFWcursor*> Cursors = {
+            { WindowDelegate::Cursor::Arrow,
+                glfwCreateStandardCursor(GLFW_ARROW_CURSOR) },
```

Comment:
Do we not want to do this here?  (the way we would have done it everywhere else)
```suggestion
            { 
                WindowDelegate::Cursor::Arrow,
                glfwCreateStandardCursor(GLFW_ARROW_CURSOR)
            },
```

The current line-breaking Is a bit weird to me, but I get if we wan tot save space and don't find readability of these lines that important